AEPIdentity/Sources/IdentityState.swift

```diff
@@ -346,7 +346,7 @@ class IdentityState {
         }
 
         if let ecid = identityResponse.ecid, !ecid.isEmpty {
-            let stringHint: String? = identityResponse.hint == nil ? nil : "\(String(describing: identityResponse.hint))"
+            let stringHint: String? = identityResponse.hint == nil ? nil : "\(String(describing: identityResponse.hint!))"
```

Comment:
is it safe to force unwrap?

Reply:
Yeah, in the expression we check if the hint is nil before force unwrapping.